### DISCUSSION TITLE
The occipitofrontalis muscle helps us what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#759] [Galaxy Trivia] [answered]
*answer with a GIF*

**Comment by peterbaccaro** [comment]
![Frontalis_muscle_animation_small](https://github.com/ParanoidUser/yolo/assets/74666079/ebbe55a9-5f1b-4230-9fc7-ff6c77e0b5cb)

**Reply by ParanoidUser** [reply]
**View image:** The occipitofrontalis muscle helps us *to spin out heads around the vertical axis*? 🤥🤭

**Reply by peterbaccaro** [reply, marked as answer]
🤣 I was too focused on the GIF part that I forgot to answer the question.

The occipital part of the occipitofrontalis muscle moves the scalp forwards, and the frontalis part lifts the brows and moves the anterior scalp backward. When the frontalis muscle contracts, the vertical fibers pull the skin of the eyebrows upward.
